server.js

```diff
@@ -1,8 +1,13 @@
-require("dotenv").config();
-const express = require("express");
+require('dotenv').config();
+const express = require('express');
+
 const app = express();
 
-app.use(express.static(__dirname + "/dist"));
+app.use(express.static(`${__dirname}/dist`));
+
+app.get('/', (req, res) => {
```

Comment:
Сейчас по пути /registry сервер ничего не отдает, нужно добавить
```suggestion
app.get('/*', (req, res) => {
```